README.md

```diff
@@ -53,3 +47,32 @@ If the data validates, you should get
   sh:conforms  true
 ] .
 ```
+
+### Test Suit
+
+The python Unittest framework uses [pySHACL](https://github.com/RDFLib/pySHACL) library for SHACL validations. 
+To install the requirements, run:
+
+```
+$ make install-requirements
+```
+
+The unit tests can be executed standalone in a python environment or using [Bazel](https://bazel.build/) build tools. 
+If you would like to run the tests using bazel, install bazel build tools with
+ 
+```
+$ npm install -g @bazel/bazelisk
```

Comment:
Unfortunately, I could not install bazel with it

Reply:
try `brew install bazel`?

Reply:
Yes, that what then worked for me @BalduinLandolt :-)

Reply:
my bold suspicion would be that you don't have npm installed ;-)

Reply:
@SepidehAlassi that might actually be something to keep in mind in these kinds of readmes: Unless you work with javascript, you may not have npm installed and the DaSCH docs recommend homebrew for installing stuff. So for future reference, it might be best to mention both installing options.